tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -1131,3 +1135,168 @@ def test_positive_provision_rhev_with_host_group(self):
 
         :Caseautomation: notautomated
         """
+
+    @upgrade
+    @run_only_on('sat')
+    @tier3
+    def test_positive_check_provisioned_rhev_os(self):
+        """Provision a host on rhev compute resource and check the hypervisor
+        provisioned VM os type.
+
+        :id: 97f66fca-50b4-42bc-b187-3b846f03ab76
+
+        :steps:
+
+            1. Prepare an RHEV  compute resource.
+            2. Create an image.
+            3. Select the custom compute profile" with custom disk size, cpu
+               count and memory.
+            4. Go to "Hosts --> New host".
+            5. Provision the host using the compute profile.
+            6. Check RHEV hypervisor VM OS type.
+
+        :expectedresults: Hypervisor VM OS type must be the same as the one
+            used in provisioning.
+
+        :CaseAutomation: Automated
+
+        :BZ: 1315281
+
+        :CaseLevel: System
+        """
+        host_name = gen_string('alpha', 9).lower()
```

Comment:
any necessity to specify all these strings length?

Reply:
No, but I can change that, if you insist

Reply:
only if you are going to have any other modification in the code